### PR TITLE
add sizzy: The browser for developers and designers

### PR DESCRIPTION
**Pull request by dz85**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/sizzy.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'sizzy' do
+  version :latest
+  sha256 :no_check
+
+  url "https://sizzy.co/get-app"
+  name 'Sizzy'
```
Comment (vitorgalvao) [inline Casks/sizzy.rb]: Please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md): `https://headway-widget.net/widgets/JAXRr7`.

File: Casks/sizzy.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'sizzy' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/sizzy.rb]: `0.11`

File: Casks/sizzy.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/sizzy.rb]: Also needs to be set.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]
Please be sure to run the commands in the template before submission. A few of the errors would have been caught beforehand.

**Review by dz85 — COMMENTED** [review]
I've changed this one, but brew cask style sizzy still failed. why? Help me.

File: Casks/sizzy.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+cask 'sizzy' do
+  version '0.11'
+  sha256 '783242bf6652c7eac79da6d1fef3bbe86239819b8e0b7b74203203ae156ef0c8'
+
+  url "https://sizzy.co/get-app"
```
Comment (vitorgalvao) [inline Casks/sizzy.rb]: Needs to use single quotes.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
> I've changed this one, but brew cask style sizzy still failed. why?

That’s why you need to follow the template. Those commands tell you what’s wrong, and in some cases auto-fix it.

**Comment by bkbabydp** [comment]
> > I've changed this one, but brew cask style sizzy still failed. why?
> 
> That’s why you need to follow the template. Those commands tell you what’s wrong, and in some cases auto-fix it.

Sorry and thx, let me try the template :)

**Comment by bkbabydp** [comment]
so bad!.. failed again

**Comment by bkbabydp** [comment]
> > I've changed this one, but brew cask style sizzy still failed. why?
> 
> That’s why you need to follow the template. Those commands tell you what’s wrong, and in some cases auto-fix it.

so bad!.. failed again

and another one , `brew cask info wechat`, it's outdated too. and i try to use `cask-repair` and it failed with the same error - `brew cask style wechat`. why?

do u have more efficient communication methods with u. 'cos i feel that the github comment is too inefficient.

**Comment by ran-dall** [comment]
https://github.com/Homebrew/homebrew-cask/pull/73391#issuecomment-559876422

**Pull request merged**